sktime/forecasting/chronos.py

```diff
@@ -0,0 +1,268 @@
+"""Implements Chronos forecaster."""
+
+__author__ = ["Z-Fran", "benheid"]
```

Comment:
since the library is forked in sktime, we should also add authors of the source code as we did with `timesfm`

Reply:
done